Dockerfile

```diff
@@ -1,18 +1,19 @@
-FROM alpine:3.5
+FROM golang:1.9.2-alpine3.7
```

Comment:
Why aren't we using the cookiecutter dockerfile?

Reply:
Ok i will.